neo/Ledger/TrimmedBlock.cs

```diff
@@ -26,7 +26,7 @@ public Block GetBlock(DataCache<UInt256, TransactionState> cache)
                 NextConsensus = NextConsensus,
                 Witness = Witness,
                 ConsensusData = ConsensusData,
-                Transactions = Hashes.Select(p => cache[p].Transaction).ToArray()
+                Transactions = Hashes.Skip(1).Select(p => cache[p].Transaction).ToArray()
```

Comment:
Why we don't store the first hash inside `ConsensusData`?

Reply:
Because there is no `ConsensusData` in `Header`. When we created a `TrimmedBlock` from a header, we need to store the hash of `ConsensusData`.